src/lib/helpers/flag.ts

```diff
@@ -36,3 +36,23 @@ export const dirPath = Flags.custom<DirContext>({
     return { abspath, exists };
   },
 });
+
+/*
+ * Takes a flag input that's supposed to be a JSON string and validates it.
+ */
+export const jsonStr = Flags.custom<string>({
```

Comment:
```suggestion
export const jsonStr = Flags.custom<AnyObj>({
```
I _think_ this is supposed to be a return type.